README.md

```diff
@@ -225,10 +225,9 @@ After these technical concerns, we move on to how you can think about all this i
 practice. These sections focus on scenarios common to employees and candidates, but are
 also of likely interest to founders and hiring managers:
 
-- [**Evaluating Equity Compensation**](#evaluating-equity-compensation): Whether you have equity now
-  or will in the future, it is helpful to learn *how to think about* the value of equity and
-  its tax burden.
-  We also cover whether you can [sell private stock](#can-you-sell-private-stock).
+- [**Plans and Scenarios**](#plans-and-scenarios): Whether you have equity now or will in the
```

Comment:
This description isn't quite accurate. Remove mention of the tax burden. Also, specify value of private equity (could even say 'at least for now'). More like: "In this section, we'll offer some important reminders* on how to evaluate your equity, and lay out some of the choices you can make about your equity, including selling private stock and different exercise scenarios." 

*see notes below. If the reminders remain we just need to frame them very clearly to avoid confusion!

Reply:
Plans and Scenarios *must* include tax burden in the future, and does at least in small ways currently.